components/ManifestoPage/ManifestoPage.tsx

```
@@ -21,42 +21,56 @@ const ManifestoPage = () => {
 We refuse.
 
-This is a space for process. For lineage. For the drafts that built dynasties.
+`}
+              <em><strong>In Process is a space for process. For lineage. For the drafts that built dynasties.</strong></em>
+              {`
 For the blueprints they stole, repackaged, and resold.
 For the histories that deserve permanence, not expiration dates.
 
-In Process is not content—it’s record-keeping. It’s proof. It’s legacy.
+`}
+              <em><strong>In Process is not content—it's record-keeping. It's proof. It's legacy.</strong></em>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)